barretenberg/cpp/src/barretenberg/relations/ecc_op_queue_relation.hpp

```diff
@@ -67,17 +67,17 @@ template <typename FF_> class EccOpQueueRelationImpl {
     {
         PROFILE_THIS_NAME("EccOp::accumulate");
         using Accumulator = std::tuple_element_t<0, ContainerOverSubrelations>;
-        using View = typename Accumulator::View;
-
-        auto w_1 = View(in.w_l);
-        auto w_2 = View(in.w_r);
-        auto w_3 = View(in.w_o);
-        auto w_4 = View(in.w_4);
-        auto op_wire_1 = View(in.ecc_op_wire_1);
-        auto op_wire_2 = View(in.ecc_op_wire_2);
-        auto op_wire_3 = View(in.ecc_op_wire_3);
-        auto op_wire_4 = View(in.ecc_op_wire_4);
-        auto lagrange_ecc_op = View(in.lagrange_ecc_op);
+        using MonomialAccumulator = typename Accumulator::MonomialAccumulator;
+
+        auto w_1 = Accumulator(MonomialAccumulator(in.w_l));
```

Comment:
why do we handle things differently here in comparison to other relations?

Reply:
I will add a comment explaining this